app/assets/javascripts/components/static/about_page.es6.jsx

```diff
@@ -2,69 +2,85 @@ class AboutPage extends React.Component {
 
   constructor(props) {
     super(props);
-    this.state = {
-      people: [
-        'daniel', 'richard', 'rocky',
-        'rachel', 'sahil', 'lancy'
-      ],
-    }
-  }
+    this.individuals = [
```

Comment:
is there a reason its `.erb`? and is `image_constants.es6.jsx.erb` already an existing file or something you created for this purpose?